src/test/java/core/basesyntax/validation/DataValidatorTest.java

```diff
@@ -0,0 +1,60 @@
+package core.basesyntax.validation;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+class DataValidatorTest {
+    private final DataValidator dataValidator = new DataValidator();
+
+    @Test
```

Comment:
Consider using descriptive test names following given/when/then style for better readability. For example, `givenEmptyList_whenValidate_thenReturnsTrue`.